eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -150,8 +152,10 @@ def __init__(self, track, params, **kwargs):
             self._window_end = [{"type": "relative", "offset_ms": 0}]
 
     def partition(self, partition_index, total_partitions):
-        seed = partition_index * self._params["seed"] if "seed" in self._params else None
```

Comment:
There was already seed parameter being used for setting random window_end. How often does partition called? Would I need to make it work with both seeds or can I reuse this one? Is it important that we use partition_index as multiplier?

Reply:
talked to Dimitrios offline, and partion [is called by Rally to “assign” the parameter source across multiple clients.](https://esrally.readthedocs.io/en/latest/adding_tracks.html?highlight=partition#custom-parameter-sources). I will make sure we use logic similar too previous that takes partition_index into account